libs/api/app/data-access/src/lib/entity/app-transaction.entity.ts

```diff
@@ -88,7 +87,7 @@ export class AppTransaction {
   @Field({ nullable: true })
   ua?: string
   @HideField()
-  webhooks?: AppWebhook[]
+  webhooks?: unknown[]
```

Comment:
Do we want type `unknown[]` here?

Reply:
Yep, otherwise we'll be back with a circular dep again. 

This property is used to work with the `ResolveField` (`tx.webhooks?.find`):

```ts
@Resolver(() => AppTransaction)
export class ApiWebhookAppTransactionFeatureResolver {
  @ResolveField(() => Webhook, { nullable: true })
  webhookEventIncoming(@Parent() tx: AppTransaction) {
    return tx.webhooks?.find(
      ({ direction, type }) => direction === WebhookDirection.Incoming && type === WebhookType.Event,
    )
  }
  ...
}
```